api/graphql/mutations/role.js

```diff
@@ -0,0 +1,88 @@
+const { GraphQLYogaError } = require('@graphql-yoga/node')
+
+export async function addGroupRole ({ groupId, color, name, emoji, userId }){
+  if (!userId) throw new GraphQLYogaError(`No userId passed into function`)
+
+  if (groupId && name && emoji) {
```

Comment:
I dont think emoji should be a requirement.... maybe? lets double check with Aaron

Reply:
If there is no emoji... how would anyone know someone has a badge/role? It'd be a silent role I guess

Reply:
I thought we would always display the name/text too

Reply:
except maybe in small spaces. a question for @brodeur